src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,14 @@
+package duke;
+
+/**
+ * The DukeException class which handles all the exceptions in the Duke program
+ *
+ * @author  Hope Leong
+ * @version 0.1
+ * @since   27/8/2020
+ */
+public class DukeException extends Exception{
+    public DukeException(String message){
+        super(message);
+    }
+}
```

Comment:
Not sure I can agree on this abstraction. Wouldn't this make it harder to locate the cause of exceptions?